egs/aishell/asr1/RESULTS.md

```diff
@@ -36,3 +36,21 @@ exp/train_sp_pytorch_no_patience/decode_test_beam20_emodel.acc.best_p0.0_len0.0-
 |    SPKR       |     # Snt         # Wrd     |     Corr           Sub            Del           Ins            Err         S.Err     |
 |    Sum/Avg    |     7176         104765     |     92.2           7.6            0.2           0.2            8.0          50.2     |
 ```
+
+# Ngram related 
+   - there is no RNN not ngram 
```

Comment:
This result looks promising. Have you ever tried ngram-RNNLM interpolation (joint decoding)?

Reply:
That's my recommendation, too, and @qmpzzpmq is trying now.

Reply:
I tried, but there is no result better than currently one.
I guess it takes time to tuning the decoding weight parameters.